docs/conditions.rst

```diff
@@ -274,6 +274,21 @@ You can optionally alias these attribute(s) using the **<<** operator. For examp
 | a comparison. **events** and **facts** have rule scope and are not available
 | outside of the rule. They can be used in assignments and accessing the saved
 | values in a condition or in action.
+| The above example using default assignments
+
+    .. code-block:: yaml
+
+        name: multiple conditions using default assignments
+        condition:
+          all:
+            - event.i == 0
+            - event.i == 1
+            - event.i == events.m_0.i + 2
+        action:
+          debug:
+            first: "{{events.m_0}}"
+            second: "{{events.m_1}}"
+            third: "{{events.m_2}}"
```

Comment:
```suggestion
            first: "{{ events.m_0 }}"
            second: "{{ events.m_1 }}"
            third: "{{ events.m_2 }}"
```

Minor nitpicking, but I think it would be good to follow Ansible best practices and include spaces between braces in our docs.